internal/app.go

```diff
@@ -174,6 +179,16 @@ OPTIONS:
   {{range $index, $option := .VisibleFlags}}{{if $index}}
   {{end}}{{$option}}{{end}}{{end}}
 `
+	cli.VersionPrinter = func(c *cli.Context) {
+		db.Init(utils.DefaultCacheDir())
+		metadata, _ := db.Config{}.GetMetadata()
```

Comment:
It sometimes returns an error. Even if we don't handle it, it might be better to display a warning/error message.

Reply:
Yes good point. Addressed here https://github.com/aquasecurity/trivy/pull/435/commits/e197eb99aef0614dbca420f7705398218299c7a5